packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -0,0 +1,46 @@
+.grav-c-list-cards-basic {
+  $breakpoint-1: gravy-breakpoint(medium);
+  $breakpoint-2: gravy-breakpoint(large);
+  padding: 0;
+  list-style-type: none;
+
+  @media all and (min-width: $breakpoint-1) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  > li + li {
+    margin-top: $grav-sp-xl;
+  }
+
+
+  @media all and (min-width: $breakpoint-1) {
+    > {
+      // 2 col layout
+      li {
+        display: flex;
```

Comment:
I think this `display: flex;` is redundant. The `<li>`s are the flex _items_, no?